src/ripple/protocol/Protocol.h

```diff
@@ -46,17 +46,6 @@ struct Protocol
     static int const txMaxSizeBytes = 1024 * 1024; // 1048576
 };
 
-/** A clock representing network time.
-    This measures seconds since the Ripple epoch as seen
-    by the ledger close clock.
```

Comment:
We need to make sure that the explanation in the comment is available in `NetClock`